static/js/shared/ga_events.ts

```diff
@@ -67,3 +67,45 @@ export const GA_VALUE_TOOL_CHART_OPTION_SHOW_DENSITY = "show density";
 export const GA_VALUE_TOOL_CHART_OPTION_EDIT_SOURCES = "edit sources";
 export const GA_VALUE_TOOL_CHART_OPTION_FILTER_BY_POPULATION =
   "filter by population";
+
+/**
```

Comment:
You mentioned that Carolyn wanted you to document the events in the github wiki (https://github.com/datacommonsorg/website/wiki). Did she want it documented in the code as well?

Reply:
Yeup, she wanted me to do both.